indico/modules/events/contributions/forms.py

```diff
@@ -32,3 +32,4 @@ class ContributionForm(IndicoForm):
     duration = TimeDeltaField(_("Duration"), [DataRequired()], default=timedelta(minutes=20),
                               units=('minutes', 'hours'),
                               description=_("The duration of the contribution"))
+    location_data = IndicoLocationField(_("Location"), description=_("Location"))
```

Comment:
Cause then I would manually have to create the HTML element for the description, whereas now I just update the inner text depending on the 'use default' checkbox.